src/dev/eslint/run_eslint_with_types.ts

```diff
@@ -65,7 +65,10 @@ export function runEslintWithTypes() {
       const failures = await Rx.lastValueFrom(
         Rx.from(projects).pipe(
           mergeMap(async (project) => {
-            const configFilePath = Path.resolve(project.directory, 'types.eslint.config.js');
+            const configFilePath = Path.resolve(
+              project.directory,
+              `types.eslint.config.${project.isEsm ? 'cjs' : 'js'}`
```

Comment:
> [!NOTE]  
> since the `kbn-dependency-packages` is marked as `module` and `types.eslint.config` is written as `commonjs` needed to adjust file extension

Reply:
Did you attempt to make `types.eslint.config.template.js` a `.cjs` file?

Reply:
pushed in 59285f1fcf78490d3f51957b7ca5b1b3b5d0cbb4, works fine, thanks for pointing out! will clean up `isEsm ` field for project